Tests/Test_SwitchRead.c

```diff
@@ -0,0 +1,96 @@
+#include "common.h"
+#include "config.h"
+#include <os.h>
+#include "Tasks.h"
+
+/**
+ * This file is a basic test file that spawns the readSwitches thread, 
+ * and another thread that prints those states to the console. 
+ * 
+*/
+static car_state_t CarState;
```

Comment:
What is the point of declaring this `static` if you just pass a pointer to it into literally all of your tasks?